archetypes/carbon-component-archetype/src/test/resources/projects/component/reference/src/main/java/org/test/component/GreeterImpl.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Here what happens is, when the integration-test goal of the maven archetype plugin runs, a sample project is generated from the archetype and it is compared against the reference project we have provided. 
Everything inside the reference folder belongs to the reference project against which the project generated using the archetype is compared. So GreeterTest is no different from GreeterImpl in that sense.